composer.json

```diff
@@ -30,6 +30,7 @@
         "composer/semver": "^3.0",
         "doctrine/doctrine-bundle": "^2.5.0",
         "doctrine/orm": "^2.10.0",
+        "phpunit/phpunit": "^9.6",
```

Comment:
Does `phpunit-bridge` read / respect this? We run `vendor/bin/simple-phpunit`, which downloads its own phpunit. Does this signal to download 9.6? I thought the version was done with an env var.

Reply:
good catch! added the `SYMFONY_PHPUNIT_VERSION` with `9.6` to `phpunit.xml.dist` instead....  `simple-phpunit` picks up on that locally and on ci when doing the install.